deltachat-ios/Assets.xcassets/ic_lock.imageset/Contents.json

```diff
@@ -1,23 +1,23 @@
 {
   "images" : [
     {
+      "filename" : "ic_lock_1x.png",
       "idiom" : "universal",
-      "filename" : "ic_lock_lightergrey_1x.png",
```

Comment:
what is that change about?

just a renaming? wondering as the file size changed.

Reply:
I've changed the images. They had a lot of empty space above the key icon, so that it was annoying to align the icon correctly.